Instanciador iniciado correctamente
-------------------- - 
Se realiza login de manera correcta
Se accede correctamente a Gestión de Cliente
Se accede correctamente a la sección Mis Ofertas lyntia
Nueva Oferta creada correctamente
Se han introducido correctamente los campos de la Oferta: CRM_COF0008, CLIENTE INTEGRACION, Cambio de configuración, # BizQA.
La Oferta se guarda correctamente.
***Se cumplen las condiciones de guardado de oferta correctamente
Se accede correctamente a la pestaña Fechas de la Oferta.
***Las condiciones de fechas informadas correctamente han sido OK
Se elimina la Oferta correctamente.
Se pulsa correctamente Eliminar.
Se accede correctamente a la sección Mis Ofertas lyntia
Nueva Oferta creada correctamente
Se han introducido correctamente los campos de la Oferta: CRM_COF0008, CLIENTE INTEGRACION, Cambio de configuración, # BizQA.
La Oferta se guarda y cierra correctamente.
Se busca la Oferta por parámetro: CRM_COF00082905230823.
***No se cumple la condición de Oferta guardada Correctamente.
-------------------- - 
-------------------- - 
Excepción: OpenQA.Selenium.WebDriverTimeoutException: Timed out after 45 seconds
 ---> OpenQA.Selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//label[@aria-label='En elaboración']/div"}
  (Session info: chrome=113.0.5672.127)
   at OpenQA.Selenium.WebDriver.UnpackAndThrowOnError(Response errorResponse, String commandToExecute)
   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
   at OpenQA.Selenium.WebDriver.FindElement(String mechanism, String value)
   at OpenQA.Selenium.By.<.ctor>b__11_0(ISearchContext context)
   at OpenQA.Selenium.By.FindElement(ISearchContext context)
   at OpenQA.Selenium.WebDriver.FindElement(By by)
   at SeleniumExtras.WaitHelpers.ExpectedConditions.<>c__DisplayClass19_0.<ElementToBeClickable>b__0(IWebDriver driver)
   at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition, CancellationToken token)
   --- End of inner exception stack trace ---
   at OpenQA.Selenium.Support.UI.DefaultWait`1.ThrowTimeoutException(String exceptionMessage, Exception lastException)
   at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition, CancellationToken token)
   at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition)
   at Lyntia.TestSet.Actions.CommonActions.waitElement(By locator) in C:\Users\arnau.suiles\Source\Repos\Lyntia-automation\Lyntia\TestSet\Actions\CommonActions.cs:line 262
   at Lyntia.TestSet.Actions.CommonActions.getElement(By locator) in C:\Users\arnau.suiles\Source\Repos\Lyntia-automation\Lyntia\TestSet\Actions\CommonActions.cs:line 236
   at Lyntia.TestSet.Conditions.OfertaConditions.validar_OfertaGuardadaCorrectamenteEnGrid() in C:\Users\arnau.suiles\Source\Repos\Lyntia-automation\Lyntia\TestSet\Conditions\OfertaConditions.cs:line 146
